lolcode devel 0.11.2

bfontaine opened the pull request:
I’m not sure how to do this change: #18628 updated `lolcode` from `0.10.3` to `0.11.1`, but `0.10.*` releases are stable versions while `0.11.*` ones are development versions.

mikemcquaid commented:
Hmm, I wonder too. Any thoughts here @jacknagel; are we better to leave it as-is for now and fix it when `0.11` leaves devel?

jacknagel commented:
I usually roll things back to stable when I notice, but since we've been on 0.11.x for almost two years (?) maybe we should wait to fix it?

tdsmith commented:
Yeah, at the risk of destabilizing people using lolcode in production I think `stable` can haz 0.11.2.

bfontaine commented:
> Yeah, at the risk of destabilizing people using lolcode in production I think `stable` can haz 0.11.2.

:+1:

Thanks all for your feedback, I added a comment and updated the formula to 0.11.2.

mikemcquaid commented:
Thanks @bfontaine!
